### DISCUSSION TITLE
How to convert meta information on a page

### DISCUSSION THREAD

**Discussion by overdev-l** [sergiodxa/remix-i18next#166] [Q&A] [answered]
I want to transform meta information, but can't use hooks outside of the component

**Comment by sergiodxa** [comment, marked as answer]
You need to use `getFixedT` in your loader and return the translated messages from there.

**Reply by overdev-l** [reply]
Thanks for replying to my question, I noticed that the conversion of the language is not ssr, what should I do？

**Reply by sergiodxa** [reply]
Ensure your i18next backend is being able to load your translation files. This is outside remix-i18next scope.
